NYPLCardCreator/Classes/ValidateAddressResponse.swift

```diff
@@ -33,9 +33,9 @@ final class ValidateAddressResponse {
         guard
           let JSONObject = object as? [String: AnyObject],
           let addressJSON = JSONObject["address"] as? [String: AnyObject],
-          let address = Address.addressWithJSONObject(addressJSON),
-          let cardTypeString = JSONObject["card_type"] as? String
+          let address = Address.addressWithJSONObject(addressJSON)
           else { return nil }
+        let cardTypeString = JSONObject["card_type"] as? String
```

Comment:
Why did we make this change here? I think we're supposed to always expect a response with a `"card_type"` of type `String`, so moving that out of the guard introduces the chance of a bad response from the server not being caught (as cardTypeString can now be `nil`).